testutil/singlevm/cleanup.sh

```diff
@@ -21,4 +21,7 @@ sudo docker rm -v -f ceph-demo
 sudo rm /etc/ceph/*
 sudo rm /etc/pki/ciao/controller_key.pem /etc/pki/ciao/ciao-image-key.pem
 sudo rm /etc/pki/ciao/controller_cert.pem /etc/pki/ciao/ciao-image-cacert.pem
+sudo rm /var/lib/ciao/images/4e16e743-265a-4bf2-9fd1-57ada0b28904
```

Comment:
Yeah...future PR/enhancement eventually.  In the meantime you've got the pragmatic option here.